nl_NL/LC_MESSAGES/youtube.po

```diff
@@ -4,191 +4,191 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 
 msgid "HEADER_UPLOAD_TO_YOUTUBE"
-msgstr "Upload to YouTube"
+msgstr "Upload naar YouTube"
 
 msgid "YOUTUBE"
 msgstr "YouTube"
 
 msgid "HEADER_CONFIRM_OPTIONS"
-msgstr "Confirm Options"
+msgstr "Bevestig Opties"
 
 msgid "LINK_REQUIRED_BEFORE_UPLOAD"
-msgstr "You'll need to link your YouTube channel before you can upload your Flipnote."
+msgstr "Je moet je Youtube account eerst linken voordat je Flipnotes naar Youtube kunt uploaden."
 
 msgid "YOUTUBE_ACCOUNT_VERIFICATION_PROMPT"
-msgstr "Your YouTube account is unverified. Certain features will not function correctly, such as video thumbnails and Flipnotes longer than 15 minutes."
+msgstr "Je youtube account is nog niet geverifieerd om verschillende functies te gebruiken moet je je youtube account eerst verifieren."
```

Comment:
"YouTube" should be capitalized like that. This also appears to be 1 run-on sentence, I think the sentence should end after "geverifieerd" and "Om".

Reply:
In the new string, "om" doesn't have the first letter capitalized.